exercises/levenshtein-distance/canonical-data.json

```diff
@@ -0,0 +1,65 @@
+{
+  "exercise": "levenshtein-distance",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Calculate Levenshtein distance",
+      "cases": [
+        {
+          "description": "mine_to_times",
+          "property": "distance",
+          "input": {
+            "phrase": "mine",
+			"phrase": "times"
+          },
+          "expected": "3"
+        },
+		{
+          "description": "_to_abcd",
```

Comment:
whenever possible, we should explain *why* a case is present, instead of just *what* it is. For example, this description can be improved to "insertions", since it tells us whether an implementation is capable of calculating distance for a sequence involving only insertions.